src/pool/disjoint/pool_disjoint_impl.hpp

```diff
@@ -32,26 +33,6 @@ class DisjointPoolConfig {
         std::atomic<size_t> TotalSize;
     };
 
-    // Minimum allocation size that will be requested from the system.
-    // By default this is the minimum allocation size of each memory type.
-    size_t SlabMinSize = 0;
-
-    // Allocations up to this limit will be subject to chunking/pooling
-    size_t MaxPoolableSize = 0;
-
-    // When pooling, each bucket will hold a max of 4 unfreed slabs
-    size_t Capacity = 0;
-
-    // Holds the minimum bucket size valid for allocation of a memory type.
-    // This value must be a power of 2.
-    size_t MinBucketSize = MIN_BUCKET_DEFAULT_SIZE;
```

Comment:
This is not related to this PR specifically but I just realized that we don't have any way to set the default values for params in the C API. @ldorau perhaps we should add some function to the C API, like 'umf_pool_disjoint_default_params()`?

Btw, I see that we do specify default values in https://github.com/oneapi-src/unified-memory-framework/blob/main/include/pool/pool_disjoint.h but I think this is a bug. It will not compile in C. It would be good to have at least one test for disjoint pool in C rather than in C++, so we could catch this

Reply:
Ref: https://github.com/oneapi-src/unified-memory-framework/pull/28